CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## main / unreleased
+
+### Grafana Mimir
+
+* [CHANGE] Ruler: Add `cortex_prometheus_rule_group_last_rule_duration_sum_seconds` metric to track the total evalation duration of a rule group regardless of concurrency #10189
+* [BUGFIX] Ruler: Prevent flapping (enabled/disabled) of the concurrency feature #10189
```

Comment:
```suggestion
* [BUGFIX] Ruler: Fixed an issue when using the experimental `-ruler.max-independent-rule-evaluation-concurrency` feature, where if a rule group was eligible for concurrency, it would flap between running concurrently or not based on the time it took after running concurrently. #9726 #10189
```